packages/jest/src/index.js

```diff
@@ -23,6 +21,9 @@ function getFinalPath(path) {
 }
 
 function normalizeJestOptions(opts, neutrino, usingBabel) {
+  const mediaNames = `\\.(${mediaExtensions.join('|')})`;
```

Comment:
why not `regexFromExtensions` on these ones?

Reply:
regexFromExtensions uses RegExp which returns an object. Below in the code, we have:

```js
  const moduleNameMapper = merge({
    [mediaNames]: require.resolve('./file-mock'),
    [styleNames]: require.resolve('./style-mock')
  }, moduleNames);
```

I thought that adding a RegExp object as a property would blow up (now that i think about it, i think it will just try to call its `toString` method). Also, RegExp adds a leading and a trailing slash to every expression (e.g., `/.jsx/`) which I'm not sure how Jest handles it. The options showed in the documentation in `[moduleNameMapper](https://facebook.github.io/jest/docs/en/configuration.html#modulenamemapper-object-string-string)` didn't have those leading and trailing slashes. Would it be okay to have them?

Reply:
I'm OK with leaving it how you already have it, I just wasn't for sure. Thanks for looking into that!